modules/ozoneBidAdapter.js

```diff
@@ -766,6 +1081,14 @@ export const spec = {
     ret = this._unpackVideoConfigIntoIABformat(ret, childConfig);
     return ret;
   },
+  /**
+   *
+   * look in ONE object to get video config (we need to call this multiple times, so child settings override parent)
+   * @param ret
+   * @param objConfig
+   * @return {*}
+   * @private
+   */
```

Comment:
a couple lines below this, supporting plcmt is required if you read placement from the ad unit

Reply:
@AskRupert-DM you're missing plcmt in the array below